bson/__init__.py

```diff
@@ -114,6 +114,7 @@
 from bson.son import RE_TYPE, SON
 from bson.timestamp import Timestamp
 from bson.tz_util import utc
+from bson.utc_datetime import UTCDatetimeRaw
```

Comment:
UTCDatetimeRaw is good name but I think we should discuss other options too:
- UTCDatetime
- BsonDatetime
- DatetimeMS
- TimestampMS (potentially confusing with the existing Timestamp class) 
- ?

Overall I'm leaning to DatetimeMS since that's what it represents but do you have any other ideas? We can discuss in slack too.

Reply:
Let's go with DatetimeMS since we might want to attach the CodecOptions' timezone to the DatetimeMS in the future.